releases/net471/KnownIssues/553390 - BCL - SGEN crashes because it references System.IO.Compression.ZipFile.dll.md

```diff
@@ -0,0 +1,48 @@
+# SGEN crashes trying to load the System.IO.Compression.FileSystem reference assembly
+
+## Symptoms
+
+When building an application that uses SGEN you will get an error similar to this one:
```

Comment:
- It would be good to indicate what SGEN is and include a link. So: \(XML Serializer Generator Tools (SGen.exe))\[https://docs.microsoft.com/en-us/dotnet/standard/serialization/xml-serializer-generator-tool-sgen-exe]
- comma after SGEN
- you will get --> you get